src/System.Drawing.Common/tests/GraphicsTests.cs

```diff
@@ -3460,6 +3461,30 @@ public void Clear_Disposed_ThrowsArgumentException()
             }
         }
 
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void DrawString_DefaultFont_Succeeds()
+        {
+            using (var image = new Bitmap(50, 50))
+            using (Graphics graphics = Graphics.FromImage(image))
+            {
+                graphics.DrawString("Test text", SystemFonts.DefaultFont, Brushes.White, new Point());
+                VerifyBitmapNotBlank(image);
+            }
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void DrawString_CompositingModeSourceCopy_ThrowsArgumentException()
+        {
+            using (var image = new Bitmap(10, 10))
+            using (Graphics graphics = Graphics.FromImage(image))
+            {
+                graphics.CompositingMode = CompositingMode.SourceCopy;
+                AssertExtensions.Throws<ArgumentException>(
+                    null,
+                    () => graphics.DrawString("Test text", SystemFonts.DefaultFont, Brushes.White, new Point()));
```

Comment:
Personally, I find the concept of an "empty point" a bit confusing... I think I prefer `new Point()`.